src/common/gui/CScalableBitmap.cpp

```diff
@@ -0,0 +1,63 @@
+#include "CScalableBitmap.h"
+#if MAC
+#include <CoreFoundation/CoreFoundation.h>
+#endif
+
+CScalableBitmap::CScalableBitmap( CResourceDescription desc ) : CBitmap( desc )
+{
+    int id = 0;
+    if( desc.type == CResourceDescription::kIntegerType )
+        id = (int32_t)desc.u.id;
+    
+    scales = {{ 100, 200 }};
+    scaleFilePostfixes[ 100 ] = "";
+    scaleFilePostfixes[ 200 ] = "@2x";
+
+    //fprintf( stderr, "\n" );
+    for( auto sc : scales )
+    {
```

Comment:
`for (auto sc : scales)` ?

Reply:
Weird spacing is everywhere in this commit...

Reply:
You mean "correct spacing is everywhere in this commit and the rest of the world uses weird spacing" surely ;)

That spacing will never leave my fingers. I'll clean it up when I clean up other review comments.